pkg/api/apikey.go

```diff
@@ -95,7 +95,7 @@ func (hs *HTTPServer) DeleteAPIKey(c *models.ReqContext) response.Response {
 //
 // Creates an API key.
 //
-// Will return details of the created API key
+// # Will return details of the created API key
```

Comment:
Probably should re-generate swagger due to these changes to verify it still works as expected?

Reply:
I was working with Sofia on getting this working a little bit better, but we agreed that even _if_ merging this would break Swagger it's worth getting these changes in sooner rather than later and fix it as it happens.

Unfortunately, the Go team decided to change the rules for comment formatting in a way that's rather disruptive and integrated in Go fmt, so not adhering to them leads to linter complaints. I'm not sure I think it was a bad decision seeing as Go comments have been rather lacking, but it's a nuisance.